lib/assertions/directory_content.js

```diff
@@ -0,0 +1,213 @@
+module.exports = function (chai, utils) {
+
+	var Assertion = chai.Assertion;
+	var AssertionError = chai.AssertionError;
+	var flag = utils.flag;
+	var assert = chai.assert;
+
+	var readdir = require('readdir-enhanced');
+	var format = require('util').format;
+
+	//-------------------------------------------------------------------------------------------------------------
+
+	function directoryContentAssertion (label, expectedContents, msg) {
+		var dir = flag(this, 'fs.path');
+		var deep = !!flag(this, 'deep');
+		var actualContents = flag(this, 'object');
+		var contains = !!flag(this, 'contains');
```

Comment:
No worries.  I can change it to `Boolean()`.  Neither of them is actually necessary.  I just added the `!!` to make it explicit that the `deep` and `contains` flags are booleans and the `fs.path` and `object` flags aren't